spec/routes/web/vm_spec.rb

```diff
@@ -71,6 +71,7 @@
         uncheck "Enable Public IPv4"
         choose option: "ubuntu-jammy"
         choose option: "standard-2"
+        find(".storage-slider").set(25)
```

Comment:
It's not crucial, but I believe this should be included in the next frontend commit, as I didn't see `.storage-slider` until this one.